src/Microsoft.Extensions.Caching.Abstractions/MemoryCacheExtensions.cs

```diff
@@ -119,5 +128,15 @@ public static TItem GetOrCreate<TItem>(this IMemoryCache cache, object key, Func
 
             return (TItem)result;
         }
+
+        #region Private Members
```

Comment:
No regions please.

Reply:
@Tratcher : For my learning, can you please let me know why not to use regions. I have been using them in my source code. Is that not a good practice. Should I remove them and raise a new pull-request.

Reply:
It's only a style issue, we don't use them in most cases, especially not for a single method.